Mathlib/GroupTheory/GroupAction/Hom.lean

```diff
@@ -1,212 +1,524 @@
 /-
 Copyright (c) 2020 Kenny Lau. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Kenny Lau
+Authors: Kenny Lau, Antoine Chambert-Loir
+
+! This file was ported from Lean 3 source module algebra.hom.group_action
+! leanprover-community/mathlib commit e7bab9a85e92cf46c02cb4725a7be2f04691e3a7
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
 -/
+
 import Mathlib.Algebra.GroupRingAction.Basic
 import Mathlib.Algebra.Module.Basic
-
-#align_import algebra.hom.group_action from "leanprover-community/mathlib"@"e7bab9a85e92cf46c02cb4725a7be2f04691e3a7"
+import Mathlib.Algebra.Ring.Equiv
```

Comment:
Could you please put back the `#align_import` line? That is the new way to track Mathlib 3 -> 4 alignment.

Reply:
done